src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -83,32 +125,33 @@ You can combine selectors using boolean logic.
 
 ### Value Types
 
-Select subsections by JSON-type or semantic meaning using the following:
-
-* `$string`
-* `$number`
-* `$boolean`
-* `$datetime`
-* `$array`
-* `$object`
-* `$event`
-* `$exception`
-* `$stacktrace`
-* `$frame`
-* `$request`
-* `$user`
-* `$logentry` (also applies to `event.message`)
-* `$thread`
-* `$breadcrumb`
-* `$span`
-* `$sdk`
-
-Examples:
+Select subsections by JSON-type using the following:
+
+* `$string` matches any string value
+* `$number` matches any integer or float value
+* `$datetime` matches any field in the event that represents a timestamp
```

Comment:
Would it make sense to talk about formats here?

A timestamp could be RFC3399, a numeric value (integer or float) representing time since the UNIX Epoch, or a myriad of actual formats.

Reply:
That does not pattern-match but matches against *locations* we understand as timestamp (so behind the scenes it's a set of json paths, not a regex)

Reply:
Oh, yes. And that's actually what the description says. Disregard my question.